src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -278,12 +295,38 @@ must be supplied as the value to be matched."
   (when value
     (json/parse-string (.getValue value))))
 
+(pls/defn-validated str->pgobject :- PGobject
```

Comment:
Should the older pgjson->str (and reverse) conversion methods just wrap this now perhaps? Or maybe we no longer need the pgjson code at all any more?